denso_robot.py

```diff
@@ -100,3 +113,8 @@ def check_sample_mounted(self, mount, puck_pos, pin_pos):  # is the correct samp
                     return UNMOUNT_FAILURE
         else:
             return MOUNT_STEP_SUCCESSFUL  # always successful if robot is not online
+
+# This is a generic interface to give robots access to the robot_lib api
+class OphydRobot(DensoRobot):
```

Comment:
is there any reason why you couldn't rename `DensoRobot` as `OphydRobot`? you haven't added any value by making the `OphydRobot` class

Reply:
renamed the base class and removed the interface in [c59660c](https://github.com/NSLS-II/lsdc/pull/242/commits/c59660cc9cbff23975eb43960b4703ed294bbacc)